runtime/src/bank.rs

```diff
@@ -1728,8 +1728,8 @@ impl Bank {
 
         report_new_bank_metrics(
             slot,
-            new.block_height,
             parent.slot(),
+            new.block_height,
             NewBankTimings {
```

Comment:
https://github.com/solana-labs/solana/blob/0cdb315b5b42ea276a8807271e4d8cd183ed05f3/runtime/src/bank/metrics.rs#L120-L125

Reply:
https://github.com/solana-labs/solana/pull/29628/files#r1138592620

Reply:
Ahhh, good catch / thank you for cleaning up my mistake!